build.gradle

```
@@ -343,7 +347,8 @@ distributions {
                 exclude '**/build/**', 'doc_src/**/pdf/**', 'local.properties'
             }
             into('lib/provided') {
-                from configurations.runtime
+                from configurations.implementation, configurations.api
+
```

According to [this](https://docs.gradle.org/current/userguide/java_plugin.html#tab:configurations) we can do
```suggestion
                from configurations.runtimeClasspath
```

This works in my testing.